enhancements/network/host-port-registry.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Host Port Registry
+authors:
+  - "@squeed"
+reviewers:
+  - "@dcbw"
+  - "@jsafrane"
+  - "@
+approvers:
+  - "@smarterclayton"
+creation-date: 2020-08-26 
+last-updated: 2020-08-26
+status: implementable
```

Comment:
I think implemented is fine.  I'm also proposing an `informational` status in #453 which I feel would be appropriate for this doc, too.